Readme.md

```diff
@@ -191,6 +192,57 @@ while (keys.length) {
 
 [crockfordconvention]: http://javascript.crockford.com/code.html
 
+## Declare Classes
+
+Declare a class using the `@constructor` jsdoc annotation. Like,
```

Comment:
I cannot find the exact article on the performance improvements annotations has on Chrome's V8 engines. But [here](https://developers.google.com/v8/experiments?hl=en) is another article that talks about how folks working on V8 are thinking about introducing another "Strong Mode" to improve the language performance.

In short, I think annotations are are must-have's if not as much as diligently possible. Maybe you can elaborate a bit more on where the disagreement comes from? If it's just styling issue, I think we just need to get pass that. Modern editors like WebStorm does a very good job understanding JsDoc and making code much more readable.